media-libs/opencolorio/patches/opencolorio-2.3.2.patchset

```diff
@@ -0,0 +1,33 @@
+From 1f819ff0de3677363763278ec265180bf4f7546b Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Wed, 19 Jun 2024 14:17:46 +0200
+Subject: Fix build (hackish)
+
+
+diff --git a/src/utils/NumberUtils.h b/src/utils/NumberUtils.h
+index 08a9ba0..eec81fe 100644
+--- a/src/utils/NumberUtils.h
++++ b/src/utils/NumberUtils.h
+@@ -64,6 +64,9 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
+     double
+ #ifdef _WIN32
+     tempval = _strtod_l(first, &endptr, loc.local);
++#endif
++#ifdef __HAIKU__
++    tempval = ::strtod(first, &endptr);
+ #else
+     tempval = ::strtod_l(first, &endptr, loc.local);
+ #endif
+@@ -108,6 +111,9 @@ really_inline from_chars_result from_chars(const char *first, const char *last,
+ #elif __APPLE__
+     // On OSX, strtod_l is for some reason drastically faster than strtof_l.
+     tempval = static_cast<float>(::strtod_l(first, &endptr, loc.local));
++#endif
++#ifdef __HAIKU__
```

Comment:
I think this would be more correct in this case:

```suggestion
+#elif defined(__HAIKU__)
```

for the Haiku case it wouldn't matter, but for the non-Haiku case ;)

Reply:
Need to check, I think I tried that approach ... :)